src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -150,6 +150,22 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
         }
     }
 
+    /**
+     * Changes the theme of NeoBook to dark mode.
+     */
+    @FXML
+    public void handleDarkMode() {
+        toggleTheme(false, false);
+    }
+
+    /**
+     * Changes the theme of NeoBook to light mode.
+     */
+    @FXML
+    public void handleLightMode() {
+        toggleTheme(true, false);
+    }
```

Comment:
Just FYI, `@FXML` annotations are used to expose the methods for components in the `fxml` file to reference directly, so they aren't necessary if the methods are only used in the class.